styles/src/base_layers.ts

```diff
@@ -594,6 +594,7 @@ export function nolabels_layers(
       type: "fill",
       source: source,
       "source-layer": "buildings",
+      filter: ["!=", "kind", "address"],
```

Comment:
I think this should be an opt-in instead of an opt-out, e.g. `kind` in `building` or `building_part`